### ISSUE TITLE
Message claims that secondary indexes are not dumped

### ISSUE THREAD

**Issue by AtnNn** [rethinkdb/rethinkdb#3708]
```
$ rethinkdb-dump
NOTE: 'rethinkdb-dump' only dumps data and does *not* dump secondary indexes or
 cluster metadata.  You will need to recreate your secondary indexes and cluster
 setup yourself after you run 'rethinkdb-restore'.
```

**Comment by Tryneus** [comment]
The fix is up in review 2560.  Changed the message to mention that the dump includes secondary indexes.  Also included output in the export script, the line `5422 rows exported from 3 tables` is now `5422 rows exported from 3 tables, with 2 secondary indexes`.

**Comment by Tryneus** [comment]
This has been approved and merged to `next` in commit 4587199, and cherry-picked into `v1.16.x` in commit 4442d7d.  Since this is a python driver change and non-critical (log statements only) - it will probably not be released until 1.16.1, but we'll see.
